trackers_integration/issuetracker/mantis.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+import requests
+
+from tcms.core.contrib.linkreference.models import LinkReference
+from tcms.core.templatetags.extra_filters import markdown2html
+from tcms.issuetracker.base import IntegrationThread, IssueTrackerType
+
+
+class MantisAPI:
+    """
+    Mantis Rest API interaction class.
+
+    :meta private:
+    """
+
+    def __init__(self, base_url=None, password=None, api_url=None):
+        self.headers = {
+            "Accept": "application/json-patch+json",
+            "Content-type": "application/json-patch+json",
+            "Authorization": password,
```

Comment:
FTR this will need to change to using API tokens. I'm not sure if it is just a matter of using the token instead of a password or if any other headers need to be updated.